web/ts/watch.ts

```diff
@@ -26,6 +26,10 @@ function startWebsocket() {
     if (cf !== null && cf != undefined) {
         (document.getElementById("chatForm") as HTMLFormElement).addEventListener("submit", (e) => submitChat(e));
     }
+    ws.onopen = function (e) {
+        hideDisconnectMsg();
+    };
```

Comment:
I'd initialize this hidden so the red message doesn't flicker up every time a page is opened.